code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1892,6 +1893,96 @@ GLOBAL_LIST_EMPTY(roundstart_races)
+					humi.adjust_bodytemperature(max(- gas_amount * 0.25, BODYTEMP_COOLING_MAX) * delta_time)
+					if(humi.bodytemperature < 265)
+						humi.apply_status_effect(/datum/status_effect/freon)
+			if(/datum/gas/halon)
+				if(gas_amount > 15)
+					humi.apply_status_effect(/datum/status_effect/stoned)
+			if(/datum/gas/healium)
+				if(gas_amount > 5)
+					to_chat(humi, span_notice("You feel better."))
+					humi.adjustBruteLoss(- gas_amount * 0.05 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.015 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.015 * delta_time)
+			if(/datum/gas/helium)
+				if(gas_amount > 15)
+					humi.apply_status_effect(/datum/status_effect/floaty)
+			if(/datum/gas/hypernoblium)
+				if(gas_amount > 5)
+					humi.apply_status_effect(/datum/status_effect/antimagic)
+			if(/datum/gas/nitryl)
+				if(ispodperson(humi))
+					humi.adjustToxLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					var/current_reagent = humi.reagents.get_reagent_amount(/datum/reagent/nitryl)
+					humi.reagents.add_reagent(/datum/reagent/nitryl, max(0, 1 - current_reagent))
+					continue
+				if(gas_amount > 5 && gas_amount < 10)
+					to_chat(humi, span_warning(pick("Your skin feels itchy.", "You feel like burning.")))
+				else if(gas_amount >= 10)
+					to_chat(humi, span_warning(pick("Your skin feels hot.", "You feel like burning.")))
+					humi.adjustFireLoss(gas_amount * 0.05 * delta_time)
+			if(/datum/gas/oxygen)
+				if(gas_amount > 50 && environment.temperature > 400)
+					humi.adjust_fire_stacks(gas_amount * 0.1)
+			if(/datum/gas/plasma)
+				if(isplasmaman(humi))
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.15 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/pluoxium)
+				if(gas_amount > 10)
+					humi.adjustOxyLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/proto_nitrate)
+				if(gas_amount > 10)
+					to_chat(humi, span_warning(pick("Your skin burns!", "You want to remove your skin!")))
+					humi.apply_status_effect(/datum/status_effect/gas_rash)
+			if(/datum/gas/tritium)
+				if(gas_amount > 1 && gas_amount < 5)
+					to_chat(humi, span_warning(pick("Your skin feels tingly.", "You feel tingly all over.")))
+				else if(gas_amount >= 5)
+					to_chat(humi, span_warning(pick("Your skin burns!", "You feel weak!")))
+					humi.radiation += gas_amount * 0.15 * delta_time
+					var/current_stamina = humi.getStaminaLoss()
+					if(current_stamina > 25)
+						humi.adjustStaminaLoss(gas_amount * 0.15 * delta_time)
+			if(/datum/gas/water_vapor)
+				if(humi.wear_mask?.clothing_flags & BLOCK_GAS_SMOKE_EFFECT || humi.wear_mask?.clothing_flags & MASKCOVERSEYES || humi.head?.flags_cover & HEADCOVERSEYES)
+					continue
+				if(gas_amount < 5)
+					continue
+				humi.apply_status_effect(/datum/status_effect/gas_fog, gas_id)
+			if(/datum/gas/zauker)
```

Comment:
deadly gas that kills you

Reply:
I don't understand this, outside of caustic gas is caustic

Reply:
that's it, caustic gas is caustic, nothing more, is a dangerous gas and every spill should be very dangerous

Reply:
good enough